.travis.yml

```diff
@@ -3,34 +3,35 @@ language:
 - java
 - node_js
 node_js:
-- '7'
+- '10'
+services:
+- docker
 jdk:
 - openjdk11
 addons:
   sauce_connect: true
-  apt:
-    packages:
-      - oracle-java8-installer
 env:
 - TERM=dumb
 install:
 - cd web && npm run init
 - cd ../bundles && ./gradlew --version
+- cd ..
 script:
-- cd ../web && npm run build-prod
-- cd ../bundles && ./gradlew --no-daemon build export --continue
-#- cd ../bundles/specmate-std-env/generated/distributions/executable/ && java -Xmx6000m -Djdk.crypto.KeyAgreement.legacyKDF=true -jar dev-specmate-all.jar &
-#- sleep 10
-#- cd ../ui-tests/ && mvn clean test
-after_success:
-- git status
-- echo $TRAVIS_TAG
-- cd bundles/specmate-std-env/generated/distributions/executable/
-- mv dev-specmate-all.jar dev-specmate-all-$TRAVIS_TAG.jar 
-- mv prod-specmate-all.jar prod-specmate-all-$TRAVIS_TAG.jar
-- mv prod-specmate-cdo-server.jar prod-specmate-cdo-server-$TRAVIS_TAG.jar
-- mv prod-specmate-no-cdo-server.jar prod-specmate-no-cdo-server-$TRAVIS_TAG.jar
-- cd ../../../../..
+- (cd web && npm run build-prod)
+- (cd bundles && ./gradlew --no-daemon build --continue -x check)
+- (cd bundles && ./gradlew --no-daemon export -x check)
+# - (cd bundles/specmate-std-env/generated/distributions/executable/ && java -Xmx6000m -Djdk.crypto.KeyAgreement.legacyKDF=true -jar dev-specmate-all.jar &)

```

Comment:
remove the uncommented lines, or is there a reason you keep them?

Reply:
This disables the UI-Tests (because they were unstable). That's why we should keep them.

Reply:
ok!